rasa/nlu/selectors/response_selector.py

```diff
@@ -366,9 +387,13 @@ def process(self, message: Message, **kwargs: Any) -> None:
         )
 
         prediction_dict = {
-            "response": self._full_response(label) or {TEXT: label.get("name")},
+            "response": {
+                "id": label["id"],
```

Comment:
just for my understanding, what exactly is the id of label?

Reply:
It's the hash of whatever is stored in the `name` of label.